src/wcs2_0/src/main/java/org/geoserver/wcs2_0/response/WCS20GetCoverageResponse.java

```diff
@@ -34,7 +35,7 @@
     CoverageResponseDelegateFinder responseFactory;
 
     public WCS20GetCoverageResponse(CoverageResponseDelegateFinder responseFactory) {
-        super(GridCoverage.class);
+        super(GridCoverage.class, new LinkedHashSet<>(responseFactory.getOutputFormats()));
```

Comment:
Looks to be related, but a different issue being addressed?

Reply:
Indeed it's part of its own commit which refers to its own jia ticket: "[GEOS-10156] WCS20GetCoverageResponse does not advertise the list of supported MIME types"

Reply:
My bad - missed the second commit.